packages/calcite-components/src/utils/focusTrapComponent.ts

```diff
@@ -10,12 +10,8 @@ export interface FocusTrapComponent {
   /** When `true`, prevents focus trapping. */
   focusTrapDisabled?: boolean;
 
-  /**
-   * When defined, provides a condition for focusTrapDisabled override.
-   *
-   * @internal
-   */
-  _focusTrapDisabledOverride?: () => boolean;
+  /** When defined, provides a condition for focusTrapDisabled override. */
+  focusTrapDisabledOverride?: () => boolean;
```

Comment:
Should we note that returning true does focus trap?

Reply:
`/** When defined, provides a condition for focusTrapDisabled override. When `true`, enables focus trapping. */`
How does this look, @geospatialem?

Reply:
/** When defined, provides a condition to disable focus trapping. When `true`, prevents focus trapping. */
  focusTrapDisabledOverride?: () => boolean;

@geospatialem could you check me on the doc?

Reply:
I think we should also add tests to focusTrapComponent for testing `focusTrapDisabledOverride` to make sure it functions correctly.

Reply:
> /** When defined, provides a condition to disable focus trapping. When `true`, prevents focus trapping. */   focusTrapDisabledOverride?: () => boolean;
> 
> @geospatialem could you check me on the doc?

@Elijbet Since this is a utility it won't be present in the public doc. Defer to Matt and your expertise since it'll serve as doc for the team - that said a potential recommendation for consideration:

```
/** When `true`, a condition allows focus trapping and overrides `focusTrapDisabled`. */
```

Reply:
Adding a spec test.